blog(embertimes105): emberfest cfp reminder (#176)
ember-learn/ember-blog: source/2019-07-05-the-ember-times-issue-105.md

RFC to **support populating head tag** ⛑️,

> RFC to add `load` hook to Route 🎣

Reviewer: Not related to your PR, Chris, but can we add commas to end of the lines 15 & 16 to separate items? Thanks.

Author: added!

---

## [Reminder: EmberFest CFP Deadline August 1st 🎤](https://cfp.emberfest.eu/events/emberfest-2019)

A quick reminder that the [EmberFest](https://emberfest.eu/) CFP deadline is fast approaching. We are just short of a month away from the August 1st deadline, so get your talk in today!

> Talks are 30 minutes long ranging from in-depth technical talks along with broader talks covering other aspects of software development.

Reviewer: ```suggestion
Talks are 30 minutes long and can range from in-depth technical talks to broader talks covering other aspects of software development.
```
